codeowners/src/codeowners.rs

```diff
@@ -68,6 +69,27 @@ impl CodeOwners {
             }
         })
     }
+
+    pub fn parse(codeowners: Vec<u8>) -> Self {
+        let owners_result = GitHubOwners::from_reader(codeowners.as_slice())
+            .map(Owners::GitHubOwners)
+            .or_else(|_| {
+                GitLabOwners::from_reader(codeowners.as_slice()).map(Owners::GitLabOwners)
+            });
+
+        if let Err(ref err) = owners_result {
+            log::error!(
+                "Found CODEOWNERS file `{}` in bundle, but couldn't parse it",
+                err
+            );
+        }
```

Comment:
This isn't part of the CLI, so I'd remove this. This will just log into the void.